src/include/api/ivf_pq_index.h

```diff
@@ -573,7 +573,7 @@ class IndexIVFPQ {
     }
 
     uint64_t nlist() const override {
```

Comment:
Should we also rename the API level function `nlist->partitions`?

Reply:
Sure, will do this in a follow-up PR!